starter_code/src/movies.js

```diff
@@ -93,25 +93,28 @@ ratesAverage(movies);
 
 
 // Iteration 5: Drama movies - Get the average of Drama Movies
+//Can't solve more. Getting errors. Can't figure out.
+function dramaMoviesRate(arr) {
+  const sortDrama = arr.filter(function(e) {
+    return e.genre.includes("Drama");
+  });
 
```

Comment:
here you can return 0 if the drama movies array (sortDrama) is empty:
if(!sortDrama.length) return 0;